Tasks/PublishBuildArtifacts/publishbuildartifacts.ts

```diff
@@ -5,6 +5,14 @@ import tl = require('vsts-task-lib/task');
 
```

Comment:
When you build, isn't a .js file supposed to be generated as well? Please commit that as well if you forgot.